build.gradle

```diff
@@ -21,9 +21,9 @@ subprojects {
 
 allprojects {
     repositories {
-        jcenter()
+        google()
```

Comment:
why are we adding this?

Reply:
https://blog.gradle.org/jcenter-shutdown

Reply:
I know this, I'm talking about the `google()` addition.